server/@types/express/index.d.ts

```diff
@@ -45,7 +47,8 @@ export declare global {
       verified?: boolean
       id: string
       logout(done: (err: unknown) => void): void
-      flash(type: string, message: string | ErrorMessages | Array<ErrorSummary> | Record<string, unknown>): number
+      flash(type: string, message: FlashMessage): number
+      flash(type: string): FlashMessage
```

Comment:
what does it mean to have two methods with the same name on an interface? Are both signatures accepted in all cases?

Reply:
Yep, so `flash` can be called with either of those signature. Shows up in VS code as this
![image](https://user-images.githubusercontent.com/94137563/233036824-2c19e2fe-39b7-4a07-9266-d9dced9c6abb.png)

It expands out to 6 possibilities due to `FlashMessage` being a composite type